flepimop/gempyor_pkg/src/gempyor/outcomes.py

```diff
@@ -198,6 +198,8 @@ def read_parameters_from_config(modinf: model_info.ModelInfo):
                         parameters[new_comp]["delay::npi_param_name"] = f"{new_comp}::delay".lower()
                 else:
                     logging.critical(f"No delay for outcome {new_comp}, using a 0 delay")
+                    # FIXME: We should not here modify outcomes_config := modinf.outcomes_config["outcomes"]
+                    # because this changes the order in which the outcomes the next time this function is called.
```

Comment:
We should be tracking this with an issue - possible to explain via that way? Fine with the FIXME marker here (tho @TimothyWillard  mgiht not be), but we'll lose track of it if not an issue.

Reply:
Yeah I agree. It's in https://github.com/HopkinsIDD/flepiMoP/issues/373 as the PR should not close this issue (but fixes it in Sara's use case), though perhaps redefining the title would be helpful, so it's more precise.